app/src/androidTest/java/doit/study/droid/data/local/dao/QuestionDaoTest.kt

```diff
@@ -0,0 +1,203 @@
+import android.database.sqlite.SQLiteConstraintException
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.room.Room
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.SmallTest
+import doit.study.droid.data.local.QuizDatabase
+import doit.study.droid.data.local.dao.DbTest
+import doit.study.droid.data.local.dao.util.MainCoroutineRule
+import doit.study.droid.data.local.entity.Question
+import doit.study.droid.data.local.entity.QuestionTagJoin
+import doit.study.droid.data.local.entity.Tag
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.CoreMatchers.containsString
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.contains
+import org.hamcrest.Matchers.containsInAnyOrder
+import org.hamcrest.core.IsEqual
+import org.junit.*
+import org.junit.runner.RunWith
+import java.util.*
+
+
+@ExperimentalCoroutinesApi
+@RunWith(AndroidJUnit4::class)
+@SmallTest
+class QuestionDaoTest: DbTest() {
+    @Test
+    fun insertQuestionAndGetById() = runBlockingTest {
```

Comment:
I think you can use the similar approach for naming as for unit tests. It would be better to use the same style.

Reply:
@rinekri we cannot use the same naming (backticked function name) for instrumentation tests.